spec/proforma/exporter_spec.rb

```diff
@@ -398,6 +398,73 @@
         it 'adds correct version-attribute to node' do
           expect(unittest_node.attribute('version').text).to eql '4.10'
         end
+
+        context 'with multiple custom data entries' do
+          let(:configuration) do
```

Comment:
This configuration already appears twice in this commit, right? Let's move it to a factory, so that we can simply reuse it :)